README.md

```diff
@@ -277,10 +277,12 @@ a given service on cloud environments via APIs that leverage the
 Intersmash makes this feature available for currently supported products (see the table below), but that can be 
 extended easily, since Intersmash _provisioners_ are pluggable components.
 
-| Product    | Supported Operator version | Channel name | Supported product version | Repository                                        | Notes                                                   |
-|:-----------|:---------------------------|:-------------|:--------------------------|:--------------------------------------------------|:--------------------------------------------------------|
-| Infinispan | 2.3.1                      | 2.3.x        | 14.0.6.Final              | https://github.com/infinispan/infinispan-operator |                                                         |
-| WildFly    | 0.5.6                      | alpha        | 27.0.1.Final              | https://github.com/wildfly/wildfly-operator       | As available on https://operatorhub.io/operator/wildfly |
+| Product                   | Supported Operator version | Channel name   | Supported product version | Repository                                        | Notes                                                                               |
+|:--------------------------|:---------------------------|:---------------|:--------------------------|:--------------------------------------------------|:------------------------------------------------------------------------------------|
+| Hyperfoil                 | 0.21.0                     | alpha          | 0.23                      | https://github.com/Hyperfoil/hyperfoil-operator   |                                                                                     |
```

Comment:
Is this line supposed to be part of this MR?

Reply:
Yes, I know. I took the chance and added it because was forgotten, so I isolated it in one commit: https://github.com/Intersmash/intersmash/pull/35/commits/54e5a050093cc7fbb2127d9e433f165a85bc87fb

Reply:
okay then :)